plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -215,29 +235,54 @@ var FbListInlineEdit = new Class({
 				this.select(e, this.editing);
 				this.cancel(e);
 			}
-
 			break;
 		case 13:
 			//enter
-			
 			// Already editing or no cell selected
-			if (this.inedit || typeOf(this.td) !== 'element') {
+			if (typeOf(this.td) !== 'element') {
 				return;
 			}
-			e.stop();
-			if (typeOf(this.editing) === 'element') {
+			if (this.inedit && typeOf(this.editing) === 'element') {
 				// stop textarea elements from submitting when you press enter
 				if (this.editors[this.activeElementId].contains('<textarea')) {
 					return;
 				}
+				e.stop();				
 				this.save(e, this.editing);
+				this.select(e, this.getNextEditable(this.td));
 			} else {
 				this.edit(e, this.td);
 			}
 			break;
+		default:
+			break;
 		}
 	},
-
+	
+	downaction: function(e,row) {
```

Comment:
Assuming you meant space after commas - OK done, in numerous places. (I think I got them all)